project_templates/fastapi_safir_app/{{cookiecutter.name}}/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Change log
+
+All notable changes to {{cookiecutter.name}} will be documented in this file.
+
+Versioning follows [semver](https://semver.org/).
+
+Dependencies are updated to the latest available version during each release. Those changes are not noted here explicitly.
+
+This project uses [scriv](https://scriv.readthedocs.io/en/stable/) to maintain the change log.
+Changes for the upcoming release can be found in [changelog.d](https://github.com/{{cookiecutter.github_org}}/{{cookiecutter.name}}/tree/main/changelog.d/).
```

Comment:
This preamble seems a bit long. Here's a possible edit:

```suggestion
{{cookiecutter.name}} is versioned with [semver](https://semver.org/).
Dependencies are updated to the latest available version during each release, and aren't noted here.

Find changes for the upcoming release in the project's [changelog.d directory](https://github.com/{{cookiecutter.github_org}}/{{cookiecutter.name}}/tree/main/changelog.d/).
```

If we want to include directions to developers for updating the changelog and mention Scriv we can do that with HTML/Markdown comments, but I don't think we need to specify that in the published document.